MNPL-54 | Add move logic

MerakseS opened the pull request.

MerakseS committed:
MNPL-54 | Add move logic

Casper133 commented on src/main/services/game/GameLoopService.ts:
```diff
@@ -5,22 +5,35 @@ import ClientError from '../../models/error/ClientError';
+  return Math.floor(Math.random() * (MAX_MINUS_MIN + 1)) + MIN_DICE_NUMBER;
 }
 
+// function testBalanceChange(gameState: GameStateDto) {
+//   const randomBalanceDelta = Math.floor(Math.random() * 201) - 100;
+//   PlayerBalanceChangeEventDispatcher.dispatchEvent(
+//     gameState,
+//     gameState.players[0].playerId,
+//     randomBalanceDelta
+//   );
+// }
```
я вчера сделал коммит, который убрал этот код, из-за этого у тебя сейчас конфликты

MerakseS committed:
MNPL-54 | Resolve merge conflicts with develop

MerakseS committed:
MNPL-54 | Add cellsList skeleton

MerakseS committed:
MNPL-54 | Add property and taxes logic

MerakseS committed:
MNPL-54 | Add chance logic

IdusMartiae committed:
Implement START_OF_THE_GAME event
- Add event to the event enum
- Implement event dispatcher
- Add event dispatcher call in the start() method of GameLoopService.ts

IdusMartiae committed:
MNPL-54
- Add description to the rest of cells
- Add description to the rest of chance fields
- Refactor isUserNotInGame method
